app/views/settings/index.html.erb

```diff
@@ -20,6 +20,9 @@
           </div>
         <% end %>
       </div>
+	  <% if current_user.uid.nil? %>
```

Comment:
You can write this as <% unless current_user.uid %> and it works as well.  nil checks always seem ugly to me